src/server/scripts/Spells/spell_item.cpp

```diff
@@ -3715,35 +3715,36 @@ class spell_item_taunt_flag_targeting : public SpellScript
     }
 };
 
+// 29830 - Mirren's Drinking Hat
```

Comment:
it need be where was before xd

Reply:
Oh. I did not know that. Why, though?
Isn't it supposed to be a section marker?